examples/display_camera/sdkconfig.bsp.esp32_s3_korvo_2

```diff
@@ -2,11 +2,15 @@
 # Espressif IoT Development Framework (ESP-IDF) Project Minimal Configuration
 #
 CONFIG_IDF_TARGET="esp32s3"
+CONFIG_ESPTOOLPY_FLASHMODE_QIO=y
 CONFIG_ESPTOOLPY_FLASHSIZE_4MB=y
-CONFIG_ESP_DEFAULT_CPU_FREQ_240=y
+CONFIG_SPIRAM=y
+CONFIG_SPIRAM_MODE_OCT=y
+CONFIG_SPIRAM_FETCH_INSTRUCTIONS=y
+CONFIG_SPIRAM_SPEED_80M=y
 CONFIG_LV_COLOR_16_SWAP=y
 CONFIG_LV_MEM_CUSTOM=y
 CONFIG_LV_MEMCPY_MEMSET_STD=y
 CONFIG_LV_USE_PERF_MONITOR=y
 CONFIG_LV_SPRINTF_CUSTOM=y
-# CONFIG_LV_BUILD_EXAMPLES is not set
+CONFIG_LV_BUILD_EXAMPLES=n
```

Comment:
Why do we need this?

Reply:
Sorry, it's automatically generated by my `idf.py save-deconfig`. It can be restored.